apps/tailwind-debug-app/src/app/probes/host-style-probe.component.css

```
@@ -0,0 +1,17 @@
+@reference "../../styles.css";
```

Stylelint (17.7.0)

[error] Unexpected unknown at-rule "@reference" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)